connector_search_engine/models/se_binding.py

```diff
@@ -77,16 +79,12 @@ def create(self, vals):
         record.jobify_recompute_json()
         return record
 
-    def write(self, vals):
-        not_new = self.browse()
-        if "active" in vals and not vals["active"]:
-            not_new = self.filtered(lambda x: x.sync_state != "new")
-            new_vals = vals.copy()
-            new_vals["sync_state"] = "to_update"
-            super(SeBinding, not_new).write(new_vals)
-
-        res = super(SeBinding, self - not_new).write(vals)
-        return res
+    @api.depends("active")
+    def _compute_sync_state(self):
+        for rec in self:
+            if rec.sync_state != "new":
+                rec.sync_state = "to_update"
+                rec.date_modified = fields.Datetime.now()
```

Comment:
it seem that we miss to implement the date_modified behaviour (it's use nowhere)
Maybe the best is to also create a computed field for date_modified

```
@api.depends("sync_state")
def _compute_date_modified(self):
    for rec in self:
        rec.date_modified = fields.Datetime.now()
```